browser/main/modals/PreferencesModal/UiTab.js

```diff
@@ -12,12 +13,27 @@ class UiTab extends React.Component {
     super(props)
 
     this.state = {
-      config: props.config
+      config: props.config,
+      options: {
+        lineNumbers: true,
+        theme: props.config.editor.theme,
```

Comment:
`theme` attribute looks not necessary for `CodeMirror` component.
https://github.com/JedWatson/react-codemirror/blob/master/src/Codemirror.js#L13-L23